src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-group/cephfs-subvolume-group.component.ts

```diff
@@ -142,7 +142,21 @@ export class CephfsSubvolumeGroupComponent implements OnInit, OnChanges {
           catchError(() => {
             this.context.error();
             return of(null);
-          })
+          }),
+          map((subvolumes: any[]) => [
+            {
+              name: '_nogroup',
+              info: {
+                mode: 1,
+                bytes_pcent: 1,
+                bytes_quota: 1,
```

Comment:
Why do you want to have mode as 1 and utilization as 1 for this? Isn't this wrong data?

We can try to ignore these and put N/A or `-` instead  
```suggestion
                mode: 1,
                bytes_pcent: 1,
                bytes_quota: 1,
```

Reply:
@nizamial09 thoughts?

Reply:
Lets avoid adding these as discusses, we can have a separate PR/tracker for this if req. Thanks__